### PR TITLE
Create antoangiaothong.akk1to.json

### PR DESCRIPTION
**Pull request by akk1to**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by akk1to** [comment]
point the domain to this repo: https://github.com/akk1to/atgtsystem (live on http://akk1to.is-a.dev/atgtsystem/)

File: domains/antoangiaothong.akk1to.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "This website is my personal portfollo XD (let it on rq)",
+  "repo": "https://github.com/akk1to/atgtsystem",
+  "owner": {
+    "username": "akk1to",
+    "email": "akk1to.dev@gmail.com",
+    "discord": "727497287777124414"
+  },
+  "record": {
+    "CNAME": "akk1to.github.io"
```
Comment (Stef-00012) [inline domains/antoangiaothong.akk1to.json:10]: (seeing https://akk1to.is-a.dev/atgtsystem/)
website should be completed first

**Review by Stef-DP — DISMISSED** [review]

**Comment by akk1to** [comment]
site completed, please process a review for the site

**Review by iostpa — APPROVED** [review]
Seems good to me

**Pull request merged**